hosts/testagent/hardware-configuration.nix

```diff
@@ -20,12 +20,12 @@
   boot.extraModulePackages = [];
 
   fileSystems."/" = {
-    device = "/dev/disk/by-uuid/b97aee02-dcbe-4755-b24c-ee1e68cbf0af";
+    device = "/dev/nvme0n1p1";
     fsType = "ext4";
```

Comment:
Seems that partitions are wrong way around. Currently /boot is on nvme0n1p1 and root is on nvme0n1p2.

Reply:
Updated.